src/renderer/scss/component/_table.scss

```diff
@@ -99,6 +99,15 @@ table.table--transactions {
   }
   td:nth-of-type(3) {
     width: 22.5%;
+
+    .btn--link {
```

Comment:
I like this tourniquitting CSS but I dislike how it is being applied. Is tourniqueting text really just a property of button links when they're inside the 3rd `td` element of a `<table.table--help>`? This seems unlikely.

If we're struggling to find a clear global rule for these, I think it makes more sense to apply turniquitting as either a table or cell level class property.

Reply:
Also, this breaks BEM